components/eventing-controller/pkg/ems2/auth/auth.go

```diff
@@ -0,0 +1,21 @@
+package auth
+
+import (
+	httpclient2 "github.com/kyma-project/kyma/components/eventing-controller/pkg/ems2/httpclient"
```

Comment:
where is httpclient1?

Reply:
is the same convention used in "ems2" marks that is a part of the same lib.